bot/cogs/snakes.py

```diff
@@ -15,33 +33,24 @@ class Snakes:
     def __init__(self, bot: AutoShardedBot):
         self.bot = bot
 
-    async def get_snek(self, name: str = None) -> Dict[str, Any]:
-        """
-        Go online and fetch information about a snake
-
-        The information includes the name of the snake, a picture of the snake, and various other pieces of info.
-        What information you get for the snake is up to you. Be creative!
+    async def get_snek(self, name: str = None) -> SnakeDef:
+        if name is not None and name.lower() == "python":
+            # return info about language
+            return SNEK_PYTHON
 
-        If "python" is given as the snake name, you should return information about the programming language, but with
-        all the information you'd provide for a real snake. Try to have some fun with this!
-
-        :param name: Optional, the name of the snake to get information for - omit for a random snake
-        :return: A dict containing information on a snake
-        """
+        # todo: find a random snek online if there name is null
+        # todo: scrape the web to find the lost sneks
 
     @command()
     async def get(self, ctx: Context, name: str = None):
-        """
```

Comment:
Also now missing a docstring. This one's important as it shows help text to the user.